scripts/benroll.js

```diff
@@ -1,16 +1,16 @@
 // Benroll
 
 module.exports = function(robot) {
-  robot.hear(/help (me|us) (ben|@benfoxall)/i, function(msg){
+  robot.hear(/help (me |us |)(ben|@benfoxall)/i, function(msg){
     msg.send("https://twitter.com/omgmog/status/608556664334233600");
   });
   robot.hear(/(palace)/i, function(msg){
-  	msg.send("[This is relevant to the conversation](http://www.bensmindpalace.co.uk/)");
+    msg.send("[This is relevant to the conversation](http://www.bensmindpalace.co.uk/)");
   });
-  robot.hear(/it's raining/i, function(msg){
-  	msg.send("It's raining and @benfoxall is still wearing shorts.");
+  robot.hear(/it('| i)s raining/i, function(msg){
+    msg.send("It's raining and @benfoxall is still wearing shorts.");
   });
-  robot.hear(/Oh yeah!/i, function(msg){
-  	msg.send("![](https://files.gitter.im/jsoxford/jsoxford.github.com/JSbn/Ben.gif)");
+  robot.hear(/Oh yeah/i, function(msg){
```

Comment:
Could be done with `/Oh yeah($|!|\.)/i`, but I thought you'd want it to pick up on a sentence like that.